nautobot/project-static/css/dark.css

```diff
@@ -25,3 +25,72 @@ html[data-theme="dark"] .opblock-summary-method {
 html[data-theme="dark"] .version {
   color: black !important;
 }
+
+/* Darker background color */
+html[data-theme="dark"] body {
+  background-color: rgb(217, 217, 217) !important;
+}
+
+/* Lighter than the background */
+html[data-theme="dark"] .navbar,
+html[data-theme="dark"] .footer {
+  background-color: rgb(209, 216, 221) !important;
+}
+
+html[data-theme="dark"] .panel,
+html[data-theme="dark"] .panel-default,
+html[data-theme="dark"] .panel-body,
+html[data-theme="dark"] .list-group,
+html[data-theme="dark"] .list-group-item,
+html[data-theme="dark"] .CodeMirror,
+html[data-theme="dark"] .modal-body {
+  background-color: rgb(222, 222, 222) !important;
+}
+
+/* Slightly lighter than background to standout */
+html[data-theme="dark"] .breadcrumb,
+html[data-theme="dark"] ul li.active a,
+html[data-theme="dark"] .panel-heading,
+html[data-theme="dark"] .panel-footer,
+html[data-theme="dark"] .model-header,
+html[data-theme="dark"] .table-headings th {
+  background-color: rgb(196, 204, 210) !important;
+}
+
+/* Form fields and backdrops, lighter than headers, navbars, etc */
+html[data-theme="dark"] #per_page,
+html[data-theme="dark"] .dropdown-menu,
+html[data-theme="dark"] .form-control,
+html[data-theme="dark"] .select2-container--default,
+html[data-theme="dark"] .select2-container--classic,
+html[data-theme="dark"] .select2-container--bootstrap,
+html[data-theme="dark"] .select2-selection--single,
+html[data-theme="dark"] .select2-selection--multiple,
+html[data-theme="dark"] .select2-search--dropdown,
+html[data-theme="dark"] .select2-search__field,
+html[data-theme="dark"] .select2-search--inline,
+html[data-theme="dark"] .select2-close-mask,
+html[data-theme="dark"] .select2-dropdown {
+  background-color: rgb(222, 222, 222) !important;
+}
+
+/* X button to close modals in top right */
+html[data-theme="dark"] .close {
+  color: rgb(243, 243, 243);
+}
+
+/* Code blocks */
+html[data-theme="dark"] pre {
+  background-color: rgb(211, 221, 222);
+  color: rgb(29, 40, 42);
+}
+
+/* Breadcrumbs */
+html[data-theme="dark"] .breadcrumb>li+li:before {
+  color: rgb(142, 142, 142);
+}
+
+/* Code */
+html[data-theme="dark"] code {
+  color: rgb(0, 255, 0);
```

Comment:
I would have preferred that this be a separate PR from the bug fix one, but won't ask you to carve it out separately at this point. Can you provide some side-by-side screenshots showing what's changed by these / why they're needed?